app/bingey-api/controllers/title.js

```diff
@@ -2,6 +2,17 @@ const { check, oneOf, validationResult } = require('express-validator');
 
 const Title = require('$/models/title');
 
+
+const getTitleById = async (req, res) => {
+  try {
+    const title = await Title.findById(req.params.id);
+
+    res.json(title);
```

Comment:
This should return the title inside a response object
```suggestion
    res.json({title: title});
```